### ISSUE TITLE
Help wanted!

### ISSUE THREAD

**Issue by piechowiakmichal** [many-passwords/many-passwords#55]
Hi!

The project is based on community contributions. The contribution is not difficult, but needed for the project to function.

The purpose of this project is to collect all the default passwords in one place.
Find out how you can contribute to this by checking the [CONTRIBUTING.md](https://github.com/many-passwords/many-passwords/blob/main/CONTRIBUTING.md) file.

Feel free to contribute!

**Comment by haidarrifki** [comment]
hi @piechowiakmichal, i just added new default password please check it out #85 if its okay, thank you!

**Comment by corv0ss** [comment]
Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.

**Comment by piechowiakmichal** [comment]
> Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.

Hi @corv0ss. Why does this seem impossible? Fork repo and made PR when you add passwords.

**Comment by piechowiakmichal** [comment]
By the way, if I can. I noticed increased traffic today. Do you know what it's source may be?

**Comment by corvoss** [comment]
> > Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.
> 
> Hi @corv0ss. Why does this seem impossible? Fork repo and made PR when you add passwords.

Done, had an issue yesterday refer to this and a problem with my account. You can close this issue, thanks. By the way, PR submitted, check #114

**Comment by corvoss** [comment]
> > Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.
> 
> Hi @corv0ss. Why does this seem impossible? Fork repo and made PR when you add passwords.

Hi. Got a problem with my account refer to this. By the way, PR opened at #114. Thanks

**Comment by piechowiakmichal** [comment]
> > > Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.
> > 
> > 
> > Hi @corv0ss. Why does this seem impossible? Fork repo and made PR when you add passwords.
> 
> Hi. Got a problem with my account refer to this. By the way, PR opened at #114. Thanks

@corvoss merged, thank you! If I can. I noticed increased traffic. Do you know what it's source may be?

**Comment by corvoss** [comment]
> > > > Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.
> > > 
> > > 
> > > Hi @corv0ss. Why does this seem impossible? Fork repo and made PR when you add passwords.
> > 
> > 
> > Hi. Got a problem with my account refer to this. By the way, PR opened at #114. Thanks
> 
> @corvoss merged, thank you! If I can. I noticed increased traffic. Do you know what it's source may be?

Nope.

**Comment by piechowiakmichal** [comment]
> > > > > Hi. Wanted to create a pull request but it seems not possible. Got a couple of passwords to add.
> > > > 
> > > > 
> > > > Hi @corv0ss. Why does this seem impossible? Fork repo and made PR when you add passwords.
> > > 
> > > 
> > > Hi. Got a problem with my account refer to this. By the way, PR opened at #114. Thanks
> > 
> > 
> > @corvoss merged, thank you! If I can. I noticed increased traffic. Do you know what it's source may be?
> 
> Nope.

Okay, thaks.

**Comment by corvoss** [comment]
PR requested at #115

**Comment by AbhishekTiwari23** [comment]
Is available for HactoberFest 2022, I want to contribute for my first PR

**Comment by piechowiakmichal** [comment]
> Is available for HactoberFest 2022, I want to contribute for my first PR

Yes, available on HacktoberFest 2022. I invite you to contribute :)

**Comment by haidarrifki** [comment]
hi @piechowiakmichal , i just updated new default password #124

**Comment by MuhammadAakash** [comment]
Hi @piechowiakmichal,
I have added some default passwords, Please review #126 PR

**Comment by mh31-priyanshu** [comment]
@piechowiakmichal can you check my pull request #132 This was my first ever contribution on github :)

**Comment by Foo-Maker** [comment]
Hi @piechowiakmichal . At #150 i have add an Cisco device.

**Comment by Ukeme-Edet** [comment]
@piechowiakmichal I just opened my first pull request, couldn't be reviewed... It's my first on GitHub at #156

**Comment by Rahul-Dabholkar** [comment]
@piechowiakmichal submitted PR #161 . Kindly check my PR

**Comment by shadan1234** [comment]
Hey can you assign this issue to me, i will be more than happy to help!!!!

**Comment by piechowiakmichal** [comment]
> Hey can you assign this issue to me, i will be more than happy to help!!!!

Hi! That's informational issue. Just open PR and feel free to contribute!

**Comment by shadan1234** [comment]
@piechowiakmichal Thanks mann for replying , i have sent a PR please review it
Again thanks mann

**Comment by m87y2k** [comment]
@piechowiakmichal submitted PR https://github.com/many-passwords/many-passwords/pull/176. First contribution to open-source :D
everything's fine?
